OneCommunityGlobal/HighestGoodNetworkApp
# EduardoV - create unit tests for the MembersAutoComplete component

# Description
Create a unit test for the MembersAutoComplete component to confirm whether it working as intended.

![Screenshot_1](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/86692035/289b0f20-edc9-445e-a921-4d5d63ef1557)

## Main changes explained:
- Export file MembersAutoComplete.jsx for the test component
- Create file MembersAutoComplete.test.js to do the unit tests

## How to test:
1. check into current branch
2. cd HighestGoodNetworkApp
3. do `npm install` to run this PR locally
4. open file src → tests → Teams → MembersAutoComplete.test.js
5. run `npm run test src/__tests__/Teams/MembersAutoComplete.test.js`

## Screenshots or videos of changes:

This is the result it's supposed to get:

![Screenshot_2](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/86692035/7057d724-ba0d-4674-8570-37dc68461bb7)

If all tests passed, everything worked as intended.
